packages/mongodb-log-writer/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@mongodb-js/mongodb-log-writer",
+  "description": "A library for writing MongoDB logv2 messages",
+  "version": "2.0.0",
+  "author": {
+    "name": "MongoDB Inc",
+    "email": "compass@mongodb.com"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://jira.mongodb.org/projects/COMPASS/issues",
+    "email": "compass@mongodb.com"
+  },
+  "homepage": "https://github.com/mongodb-js/devtools-shared",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/devtools-shared.git"
+  },
+  "keywords": [
+    "mongodb",
+    "log",
+    "logv2"
+  ],
+  "main": "dist/index.js",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./.esm-wrapper.mjs"
+    },
+    "./mongo-log-writer": "./dist/mongo-log-writer.js",
+    "./mongo-log-manager": "./dist/mongo-log-manager.js"
+  },
+  "files": [
+    "LICENSE",
+    "dist",
+    "package.json",
+    "README.md",
+    ".esm-wrapper.mjs"
+  ],
+  "scripts": {
+    "lint": "eslint src/**/*.ts",
+    "test-only": "nyc mocha --colors -r ts-node/register src/**.spec.ts",
+    "test": "npm run lint && npm run build && npm run test-only",
+    "build": "npm run compile-ts && gen-esm-wrapper . ./.esm-wrapper.mjs",
+    "prepack": "npm run build",
+    "compile-ts": "tsc -p tsconfig.json"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "bson": "^4.5.1 || ^5.0.0 || ^6.0.0",
```

Comment:
If we're doing a major release anyway, maybe we can just stick to 6.x?

Reply:
`depalign` ended up setting this to `^6.8.0` (highest devDependency version of bson in other repos), could that be too high for compatibility or is that fine?

Reply:
I don't really see how it would hurt :) 

Although it is honestly a bit annoying that depalign wants to manage this version for us. Maybe we can make this a `peerDependency`? Because the idea isn't to force bson updates on the users of this package with every new release

Reply:
It technically doesn't manage it by default (I used `--autofix` and then rolled back most of the changes as it was doing too much, this one I just left). I could set it as `peerDepedency` with `6.x` then?

Reply:
Yeah, I'd do that 👍